database/index.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/gamedb');
```

Comment:
Same note as above about using `localhost`, you probably want to use an environment variable here to store the mongodb server address so that it can be updated independently of your code per-environment when you deploy.